hawc/apps/study/templates/study/study_list.html

```diff
@@ -14,6 +14,11 @@ <h2>Available studies ({{page_obj.paginator.count}} found)</h2>
           <a class="dropdown-item" href="{% url 'riskofbias:arob_detail' assessment.pk %}">View requirements</a>
           <a class="dropdown-item" href="{% url 'riskofbias:rob_assignments' assessment.pk %}">View assignments</a>
         {% endif %}
+        {% if assessment.enable_risk_of_bias %}
+          <div class="dropdown-divider"></div>
+          <span class="dropdown-header">Study tasks</span>
+          <a class="dropdown-item" href="{% url 'mgmt:task-list' assessment.pk %}?owner={{user.id}}&status=990">View study status</a>
```

Comment:
The study page links to a user's existing task assignments page on the mgmt dashboard, where once a status is marked terminal a user's next task will start. I also implemented a similar page that only displays a user's earliest non-terminal task/status per study, and can add that back in if that was the goal.